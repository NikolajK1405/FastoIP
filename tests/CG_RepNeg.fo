fun [int] main() = let x = replicate(~2, 1) in x