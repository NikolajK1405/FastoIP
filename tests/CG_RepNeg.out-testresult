Interpreter error: Argument of "replicate" is negative: -2 at line 1, column 27
