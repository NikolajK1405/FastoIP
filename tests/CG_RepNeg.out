Runtime error at line 1: negative array size